Cargo.toml

```diff
@@ -65,6 +65,10 @@ extension-module = [ "python3-sys/extension-module" ]
 # or python3-sys. (honestly, we should probably merge both crates into 'python-sys')
 extension-module-2-7 = [ "python27-sys/extension-module" ]
 
+# These these features to explicitly control linking for Python 3.
+# (See the documentation in python3-sys/Cargo.toml for more info.)
+link-mode3-default = [ "python3-sys/link-mode-default" ]
+link-mode3-unresolved-static = [ "python3-sys/link-mode-unresolved-static" ]
```

Comment:
Just nitpicking, but I don't feel the need to include this `3` in the new feature names. This would be consistent with  `extension-module`, that does not.

Maybe it'd be better to prefix with `python` or `py`, to make explicit it's about linking Python: `py-link-mode-default`, `py-link-mode-unresolved-static`?

Reply:
I will incorporate this suggestion.